slate-video-library/src/Components/Accounts/Accountdetails.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useLoginSignupContext } from "../Context/LoginSignupContext";
+import "./Accountdetails.css";
+function Accountdetails() {
+  const { state } = useLoginSignupContext();
+  console.log(state);
```

Comment:
suggestion: remove all consoles and commented code when you push the code

Reply:
sure will do that now